client/src/components/CredentialsButton/CredentialsButton.css

```diff
@@ -0,0 +1,18 @@
+.credentials {
+  background-color: white;
+  color: #3e618f;
+  border-width: 0px;
+  border-radius: 20px;
+  box-shadow: 3px 4px #dadada;
+
+  padding: 3px 20px;
+
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 20px;
```

Comment:
For font values, I would recommend using rem rather than pixels since it can scale based on different browser font sizes.
I believe it's 16px = 1rem (on most browsers)

I've linked an article below showing why using rem is slightly better than px.
https://uxdesign.cc/why-designers-should-move-from-px-to-rem-and-how-to-do-that-in-figma-c0ea23e07a15

Additionally, rem should only be used with font-size, **don't use it** in other things such as padding/margins

Reply:
For other CSS files that use `font-size` in px please also make the appropriate changes to those as well 🙁

Reply:
Fixed: changed all px to rem using [this converter](https://nekocalc.com/px-to-rem-converter).